### PR TITLE
Allow multiple Plone instances in one Zope app

### PR DESCRIPTION
**Pull request by tomgross**
#163

### CODE REVIEW

**Comment by tisto** [comment]
@tomgross LGTM. Travis is failing though.

File: .travis.yml [hunk 1]
```diff
@@ -17,8 +17,8 @@ env:
 #    - env: PLONE_VERSION=4.3.x SOLR_VERSION=5.1.x
 before_install:
   - mkdir -p $HOME/buildout-cache/{eggs,downloads}
-  - virtualenv .
-  - bin/pip install --upgrade pip setuptools zc.buildout
+  - virtualenv -p `which python` .
+  - bin/pip install setuptools==33.1.1 zc.buildout==2.9.5
```
Comment (tisto) [inline .travis.yml]: @tomgross pinning is a good thing. I prefer to have it in requirements.txt, to make sure it is used everywhere (local dev, travis).

File: src/collective/solr/utils.py [hunk 2]
```diff
@@ -63,6 +64,10 @@ def prepareData(data):
     path = data.get('path')
     if isinstance(path, dict) and not path.get('query'):
         data.pop('path')
+    if not path in data:
```
Comment (tisto) [inline src/collective/solr/utils.py]: @tomgross I'd suggest adding a comment here why this block is necessary.

**Comment by tomgross** [comment]
@tisto The tests are always right 😉 The `prepareData` method was used in two different contexts on index and search. This is now fixed. Can you please review and merge.

This also pins _zc.buildout_ and _setuptools_ to fix travis, which is broken on master.

**Comment by tisto** [comment]
@tomgross merged. Sorry, it took so long. I plan to do a new release soon and I will push things further to have a final c.solr release in the next weeks...

**Pull request merged**